torch/_export/converter.py

```diff
@@ -397,9 +409,27 @@ def convert_graph_inputs(self):
                         persistent=True,
                     )
                 )
-                fx_node = get_node_for_param_and_buffer(
+                fx_node = get_node_as_placeholder_or_get_attr(
                     self.fx_graph, name, self.is_top_level_graph()
                 )
+            elif name in self.name_to_constant:
```

Comment:
Don't you need to check that this input is a ScriptObject?

Reply:
My reasoning is only ScriptObject will be lifted as an inputs (i.e., showing up in `ts_graph.inputs()`), so we don't really need to check? Correct me if I am wrong.